jpsreport/methods/Method_D.cpp

```diff
@@ -131,11 +131,6 @@ bool Method_D::Process(
         //------------------------------Remove peds outside geometry------------------------------------------
         for(int i = 0; i < (int) IdInFrame.size(); i++) {
             if(false == within(point_2d(round(XInFrame[i]), round(YInFrame[i])), _geoPoly)) {
-                Log->Write(
-                    "Warning:\tPedestrian at <x=%.4f, y=%.4f> is not in geometry and not "
-                    "considered in analysis!",
-                    XInFrame[i] * CMtoM,
-                    YInFrame[i] * CMtoM);
```

Comment:
Why did you remove this?

Reply:
because sometimes when the geometry does not fit the trajectories, there are tones of warnings and that's very noisy.